freud/environment.pyx

```diff
@@ -133,8 +131,7 @@ cdef class BondOrder(SpatialHistogram):
 
     @Compute._compute()
     def accumulate(self, box, points, orientations, query_points=None,
-                   query_orientations=None, str mode="bod", nlist=None,
-                   query_args=None):
+                   query_orientations=None, str mode="bod", neighbors=None):
```

Comment:
Should `mode` be a constructor argument? Probably. It wouldn't make sense to accumulate multiple frames with different modes.